components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -329,7 +307,7 @@ impl From<nsCSSPropertyID> for TransitionProperty {
                         => TransitionProperty::${prop.camel_case},
                 % else:
                     ${helpers.to_nscsspropertyid(prop.ident)}
-                        => TransitionProperty::Unsupported(Atom::from("${prop.ident}")),
+                        => TransitionProperty::Unsupported(CustomIdent(Atom::from("${prop.ident}"))),
```

Comment:
This should probably use the `atom!` macro, but perhaps not all the required atoms are registered here...